pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2011,6 +2032,15 @@ func rktRun(domainName string, ContainerImageID string, xenCfgFilename string, a
 	podUUID := strings.TrimSpace(string(uuidData))
 	log.Infof("podUUID = %s\n", podUUID)
 
+	err = lookForRktRunErrors(string(stdoutStderr))
```

Comment:
rkt is not sending xen errors while rkt run to stderr.

Reply:
I don't understand you answer.
If the errors go to stderr it would be better go get stdErr separately from stdOut and parse stdErr. And the pattern matching on stderr can be simpler since you don't need to ignore normal stdout stuff.

Reply:
https://github.com/rkt/rkt/issues/3652
In rkt, container stderr is appearing on rkt run stdout. I tried with changing the mode of the stderr and stdout from log to stream but it is also not working.